spec/ruby/core/module/fixtures/refine.rb

```diff
@@ -11,3 +11,7 @@ module IncludedModule
     def foo; "foo from included module"; end
   end
 end
+
+def build_refined_class
+  Class.new(ModuleSpecs::ClassWithFoo)
+end
```

Comment:
Could you make it a class method of `ModuleSpecs`? I'd like to avoid global methods when possible.
Another solution is to define it in `refine_spec.rb` inside the `describe`, then it's a singleton method.